thyrel-web/src/hooks/AlbumProvider.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { HolyElement } from 'types/HolyElement.type';
+import { parseJson } from '../utils/json';
+import { WebsocketEvent, WebsocketMessage } from 'types/websocket.types';
+import { useWebsocketContext } from './WebsocketProvider';
+
+type AlbumContextProps = {
+  albums?: Record<number, HolyElement[]>;
+};
+
+const AlbumContext = React.createContext<AlbumContextProps>({});
+
+type AlbumContextProviderProps = { children: React.ReactElement };
+
+export function AlbumContextProvider({ children }: AlbumContextProviderProps) {
+  const [albums, setAlbums] = React.useState<Record<number, HolyElement[]>>({});
+  const { websocket } = useWebsocketContext();
+
+  React.useEffect(() => {
+    function onMessage(event: { data: string }) {
+      const websocketMessage = parseJson<WebsocketMessage>(event.data);
+      if (!websocketMessage) return;
+
+      switch (websocketMessage.websocketEvent) {
+        case WebsocketEvent.NewAlbumElement:
+          if (!websocketMessage.album) break;
```

Comment:
```suggestion
          if (!websocketMessage.album) break;
          
```
un détail mais après une condition "inline" c'est + jolie de mettre un petit espace